code/index.html

```diff
@@ -2,21 +2,122 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Uplifting News!</title>
+    <link rel="icon" href="images/book.png" />
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Major+Mono+Display&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
```

Comment:
Tip! When selecting fonts in Google Fonts you can choose multiple fonts. Then you will only get one `<link>` to insert here

Reply:
I didn't know that, thank you!